app.py

```diff
@@ -23,14 +30,70 @@ def index():
     return render_template('pages/index.html')
 
 
-@app.route("/user_registration")
+@app.route("/user_registration", methods=["GET", "POST"])
 def user_registration():
+    if request.method == "POST":
+
+        existing_user = mongo.db.users.find_one(
+            {"full_name": request.form.get("full_name").lower()})
+
+        if existing_user:
+            flash("Username already exists")
```

Comment:
If we change to the email as a unique identifer check, then we could update this Flash to be `Email already exists` 😄